### ISSUE TITLE
Cleanup all "tagged" lines that overwrite variables when setting them because of missing response from Songbeamer

### ISSUE THREAD

**Issue by bensteUEM** [bitfocus/companion-module-songbeamer-osc#7]
At present (Songbeamer 6.0.0d) OSC does not reply the option when something is set.
Therefore a workaround to update the variable status upon change is implmented

This workround needs to be removed once Songbeamer OSC API is complete

**Comment by bensteUEM** [comment]
At present only feedback triggered by companion is updated in the variable.
If user makes changes directly in Songbeamer the variable might reflect the wrong state.

**Comment by bensteUEM** [comment]
all occurrences removed because known workarounds are no longer needed with Songbeamer 6.04a
